frontend/public/components/utils/resource-icon.tsx

```diff
@@ -4,7 +4,7 @@ import * as _ from 'lodash-es';
 
 import { K8sResourceKindReference } from '../../module/k8s';
 import { modelFor } from '../../module/k8s/k8s-models';
-import { kindToAbbr } from '../../module/k8s/get-resources';
+import { kindToAbbr } from '@console/dynamic-plugin-sdk/src/utils/k8s';
```

Comment:
@fhlavac FWIW keep in mind we will need to change these paths when we finally migrate the `resource-icon` component to the SDK.